frontend/src/utils/momentUtils.test.js

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2020 Lunatech Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {renderRangeWithYear, renderRange, weekRangeOfDate} from './momentUtils';
+import moment from 'moment';
+
+// eslint-disable-next-line
+test('renderRangeWithYear a range year with different month', () => {
+    const startDate = moment('2020-01-22');
```

Comment:
Should use `moment.utc('2020-01-22')` instead according to our choice to use moment in utc only.

Reply:
agree